Authentication/LogIn/Model/Validators.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  Validators.swift
+//  Authentication
+//
+//  Created by Daniela Riesgo on 3/7/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import Foundation
+
+
+public enum ValidationResult {
```

Comment:
Lets split all this validators in separate files inside a `Validators` folder at the root of the project next to the `Login` folder.